### DISCUSSION TITLE
Do Freetube keep video thumbnails and titles in playlist even if theyre deleted

### DISCUSSION THREAD

**Discussion by dundundunnn** [FreeTubeApp/FreeTube#5359] [Q&A]
Does Freetube keep thumbnails and titles of a video in a playlist locally even if they are deleted?  I'm worried if an uploader decide to nuke their account, I would eventually forget the videos I added to the playlist.

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
When a creator deletes a video on YT you will see the following when trying to access the playlist it was stored in:

- Thumbnail removed
- Video title
- The length of the video
- Channel name

In addition to this the video id (video url) is also stored. Even though the thumbnail is gone from FreeTube you could copy the video id and search the web and try to find it with tools like waybackmachine.
